src/block_queue.rs

```diff
@@ -93,7 +93,7 @@ impl BlockQueue {
         if let Some(&(ref block_id, _)) = self.commit_queue.front() {
             if let Some(status) = self.validator_backlog.get(block_id) {
                 if let Some(block) = status.block.clone() {
```

Comment:
This `if let Some(block) = status.block.clone()` can probably be removed since `block` doesn't get used at all with this change.